Index: index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"en\">\r\n<head>\r\n    <meta charset=\"UTF-8\">\r\n    <title>Reservia</title>\r\n    <link rel=\"stylesheet\" href=\"style.css\">\r\n</head>\r\n<body>\r\n\r\n\r\n\r\n\r\n</body>\r\n</html>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.html b/index.html
--- a/index.html	
+++ b/index.html	
@@ -1,12 +1,71 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="fr">
 <head>
+
+
     <meta charset="UTF-8">
+
     <title>Reservia</title>
+
     <link rel="stylesheet" href="style.css">
+
+
 </head>
 <body>
 
+<main>
+
+    <!-- Header -->
+
+    <header>
+        <navbar>
+
+        </navbar>
+
+    </header>
+
+    <!----- Section 1 résultat de la recherche avec les hebergement + les plus populaires --------->
+
+    <section id="result_research">
+
+        <div class="result_hebergements">
+
+        </div>
+
+        <div class="result_popular">
+
+        </div>
+
+    </section>
+
+    <!----------------- Section 2 Activités à Marseille ----------------------->
+
+    <section id="activity">
+
+
+    </section>
+
+    <!---------------- Footer ---------------->
+
+    <footer id="footer">
+
+        <div class="a_propos">
+
+        </div>
+
+        <div class="nos_hebergements">
+
+        </div>
+
+        <div class="assistance">
+
+        </div>
+
+
+    </footer>
+
+
+</main>
 
 
 
Index: style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/* Variable de couleurs */\r\n:root{\r\n    --bleu : #0065FC;\r\n    --bleuClair : #DEEBFF;\r\n    --grisFond : #F2F2F2;\r\n\r\n    /*!*Pour les couleurs*!*/\r\n    /*color: var(--bleu);*/\r\n}\r\np{\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/style.css b/style.css
--- a/style.css	
+++ b/style.css	
@@ -1,4 +1,10 @@
-/* Variable de couleurs */
+/*--------- Police d'écriture ----------*/
+@import url('https://fonts.googleapis.com/css2?family=Raleway&display=swap');
+/*
+font-family: 'Raleway', sans-serif;
+*/
+
+/*---------- Variable de couleurs ------*/
 :root{
     --bleu : #0065FC;
     --bleuClair : #DEEBFF;
